### PR TITLE
update Chinese translation

### PR DESCRIPTION
**Pull request by xinjiajuan**
The commit is messed up again, sorry, please merge it. The local has not been merged.🤣🤣

### CODE REVIEW

File: res/values-zh-rCN/yaap_strings.xml [hunk 1]
```diff
@@ -98,9 +102,24 @@
     <string name="data_usage_app_restrict_wifi_summary">开启 WiFi 数据的使用</string>
 	
     <!-- Notch: Full screen apps -->
+
```
Comment (idoybh) [inline res/values-zh-rCN/yaap_strings.xml]:
```suggestion
    <!-- Notch: Full screen apps -->
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
> The commit is messed up again, sorry, please merge it. The local has not been merged.roflrofl

git is not that hard...

File: res/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (xinjiajuan) [reply res/values-zh-rCN/yaap_strings.xml]: Keep it this way, this is a modification made when resolving local conflicts

**Review by xinjiajuan — COMMENTED** [review]
Keep it this way, this is a modification made when resolving local conflicts

**Comment by idoybh** [comment]
> Keep it this way, this is a modification made when resolving local conflicts

No. I will not. You should not resolve conflicts when submitting a translation lol
Let me fix this mess and I'll merge it

**Comment by xinjiajuan** [comment]
> > Keep it this way, this is a modification made when resolving local conflicts
> 
> No. I will not. You should not resolve conflicts when submitting a translation lol
> Let me fix this mess and I'll merge it

Okay okay, thank you, leave it to you

**Review by idoybh — APPROVED** [review]
That's better.

**Comment by idoybh** [comment]
Thank you

**Pull request merged**